worker/src/main/scala/org/apache/celeborn/service/deploy/worker/storage/StorageManager.scala

```diff
@@ -424,6 +421,20 @@ final private[worker] class StorageManager(conf: CelebornConf, workerSource: Abs
     expiredShuffleKeys.asScala.foreach { shuffleKey =>
       logInfo(s"Cleanup expired shuffle $shuffleKey.")
       if (fileInfos.containsKey(shuffleKey)) {
+        fileInfos.get(shuffleKey).asScala.foreach {
+          case (_, fileInfo) => {
+            val dir = fileInfo.getFile.getParentFile.getParentFile.getParentFile
```

Comment:
mountPoint

Reply:
now dir replace to mountPointWorkingdir, because mountPointWorkingdir is consit of mountpoint + workingdir